### PR TITLE
(maint) Add dev tools to the user profile

### PR DESCRIPTION
**Pull request by MikaelSmith**
Add some simple helpers for connecting and sending messages to the user
profile.

### CODE REVIEW

File: dev/user.clj [hunk 1]
```diff
@@ -0,0 +1,34 @@
+(ns user
+  (:require [puppetlabs.pcp.client :as pcp]
+            [puppetlabs.kitchensink.core :as ks]))
+
+(def test-parameters
+  {:server "wss://broker.example.com:8142/pcp2"
+   :cacert "/Users/michaelsmith/puppetlabs/pcp-broker/test-resources/ssl/certs/ca.pem"
+   :cert "/Users/michaelsmith/puppetlabs/pcp-broker/test-resources/ssl/certs/client01.example.com.pem"
+   :private-key "/Users/michaelsmith/puppetlabs/pcp-broker/test-resources/ssl/private_keys/client01.example.com.pem"
```
Comment (wkalt) [inline dev/user.clj]: we should make these paths relative

Comment (MikaelSmith) [inline dev/user.clj]: Oh, yes.

**Comment by MikaelSmith** [comment]
Updated.

**Comment by wkalt** [comment]
+1

**Comment by MikaelSmith** [comment]
Go ahead and merge, should be low-risk.

**Pull request merged**